electrum/gui/qt/send_tab.py

```diff
@@ -297,226 +308,158 @@ def get_frozen_balance_str(self) -> Optional[str]:
-            e.setFrozen(False)
-        for e in [self.send_button, self.save_button, self.clear_button, self.amount_e, self.fiat_send_e]:
-            e.setEnabled(True)
+            self.set_field_style(e, None, False)
+        for e in [self.save_button, self.send_button]:
+            e.setEnabled(False)
         self.window.update_status()
         run_hook('do_clear', self)
 
-    def set_onchain(self, b):
-        self._is_onchain = b
-        self.max_button.setEnabled(b)
-
-    def lock_amount(self, b: bool) -> None:
-        self.amount_e.setFrozen(b)
-        self.max_button.setEnabled(not b)
-
     def prepare_for_send_tab_network_lookup(self):
-        self.window.show_send_tab()
-        self.payto_e.disable_checks = True
-        for e in [self.payto_e, self.message_e]:
-            e.setFrozen(True)
-        self.lock_amount(True)
+        self.window.show_send_tab() # FIXME why is this here
+        #for e in [self.payto_e, self.message_e]:
+        # self.payto_e.setFrozen(True)
         for btn in [self.save_button, self.send_button, self.clear_button]:
             btn.setEnabled(False)
-        self.payto_e.setTextNoCheck(_("please wait..."))
-
-    def payment_request_ok(self):
-        pr = self.payment_request
-        if not pr:
-            return
-        invoice = Invoice.from_bip70_payreq(pr, height=0)
-        if self.wallet.get_invoice_status(invoice) == PR_PAID:
-            self.show_message("invoice already paid")
-            self.do_clear()
-            self.payment_request = None
-            return
-        self.payto_e.disable_checks = True
-        if not pr.has_expired():
-            self.payto_e.setGreen()
-        else:
-            self.payto_e.setExpired()
-        self.payto_e.setTextNoCheck(pr.get_requestor())
-        self.amount_e.setAmount(pr.get_amount())
-        self.message_e.setText(pr.get_memo())
-        self.set_onchain(True)
-        self.max_button.setEnabled(False)
-        # note: allow saving bip70 reqs, as we save them anyway when paying them
-        for btn in [self.send_button, self.clear_button, self.save_button]:
-            btn.setEnabled(True)
-        # signal to set fee
-        self.amount_e.textEdited.emit("")
+        # self.payto_e.setTextNoCheck(_("please wait..."))
 
-    def payment_request_error(self):
-        pr = self.payment_request
-        if not pr:
-            return
-        self.show_message(pr.error)
-        self.payment_request = None
+    def payment_request_error(self, error):
+        self.show_message(error)
         self.do_clear()
 
-    def on_pr(self, request: 'paymentrequest.PaymentRequest'):
-        self.payment_request = request
-        if self.payment_request.verify(self.window.contacts):
-            self.payment_request_ok_signal.emit()
+    def set_field_style(self, w, text, validated):
+        from .util import ColorScheme
+        if validated is None:
+            style = ColorScheme.LIGHTBLUE.as_stylesheet(True)
```

Comment:
try pasting an lnurl-pay into the qt payto_e, e.g.:
`LNURL1DP68GURN8GHJ7CN5VDCXZ7FWDPEXVTN0WFNJ7TNHV4KXCTTTDEHHWM30D3H82UNVWQHKSUNXLEM35E`
the `amount_e` also becomes light blue. hinting it got frozen -- but it is not frozen. in fact we expect the user to modify the amount there, I guess (?)

Reply:
Yes, the amount styling is inconsistent still, will add to top comment